spec/javascript-spec.coffee

```diff
@@ -612,6 +612,23 @@ describe "Javascript grammar", ->
       expect(tokens[6]).toEqual value: '</b>', scopes: ['source.js', 'string.quoted.template.html.js']
       expect(tokens[7]).toEqual value: '`', scopes: ['source.js', 'string.quoted.template.html.js', 'punctuation.definition.string.end.js']
 
+  describe "innerHTML attribute declarations with string template tags", ->
+    it "tokenizes them as strings", ->
+      {tokens} = grammar.tokenizeLine('text.innerHTML = `hey <b>${name}</b>`')
+      expect(tokens[0]).toEqual value: 'text', scopes: [ 'source.js', 'variable.other.object.js' ]
+      expect(tokens[1]).toEqual value: '.', scopes: [ 'source.js', 'meta.delimiter.property.period.js' ]
+      expect(tokens[2]).toEqual value: 'innerHTML', scopes: [ 'source.js', 'variable.other.property.js' ]
+      expect(tokens[3]).toEqual value: ' ', scopes: [ 'source.js', 'string.quoted.template.html.js' ]
```

Comment:
Okay, I've fixed the scopes and updated the grammar so that the output is correct.